charts/castai-agent/values.yaml

```diff
@@ -82,3 +82,7 @@ securityContext:
   fsGroup: 1002
   runAsGroup: 1002
   runAsUser: 1002
+
+cpvpa:
```

Comment:
Why can't we place it under existing `clusterVPA`? Having different keys might cause confusion. We could deprecate `clusterVPA` and use this instead because of better name (proportional is missing on the old one).